.ci/cico_do_build_che.sh

```diff
@@ -16,7 +16,7 @@ mvnche() {
   which scl 2>/dev/null
   if [ $? -eq 0 ]
   then
-    if [ `scl -l 2> /dev/null | grep rh-maven33` != "" ]
+    if [ `scl -l 2> /dev/null | grep -q rh-maven33` ]
```

Comment:
why not simply fix it with:

```
if [ "$(scl -l 2> /dev/null | grep rh-maven33)" != "" ]
```
?

I'm not a shell expert, but I'm a bit worried about what happens when the content of `scl -l 2> /dev/null | grep -q rh-maven33` would be an empty string (=> when the `scl` result doesn't contain `rh-maven33`) ?

Reply:
Your suggestion contains incorrect syntax too. So this doesn't solve the problem.
Since test command `[ ... ]` verifies return code, not string, it should work.

Reply:
Sorry, I misread your suggestion. Wrapping result of the execution with quotes would help too. I'll update the PR since this syntax seems more widely understood that grep parameter `-q`